### PR TITLE
Feature/info page moods

### PR DESCRIPTION
**Pull request by jackcasstlesjones**
# What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Styling
- [ ] Build
- [ ] Chore

## Description

- Wrote out copy for the Moods Info page with toggle to switch between content
- Used almost pure HTML as all Info pages will be fairly unique and won't have any logic, so writing reusable components is not an effective use of time
- Refactored header so that passing it `isInfoPage = true` will add a 'back' button on the top left and move the heading over to the right side on info pages

## Screenshots

Screenshot 2024-12-08 at 12 24 32
Screenshot 2024-12-08 at 12 24 40

## UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_

- [ ] Semantic HTML implemented?
- [ ] Checked with [axe DevTools](https://www.deque.com/axe/) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?

## Added/updated tests?

_Please aim to keep the code coverage percentage at 80% and above._

- [x] Yes
- [ ] No, and this is why:
- [ ] I need help with writing tests

## Delete branch after merge?

- [x] Yes
- [ ] No

## What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: src/app/moods/info/page.tsx [hunk 1]
```diff
@@ -1,3 +1,164 @@
...
+            <Image
+              src="/images/simpleEisenhower.png"
+              width={600}
+              height={600}
+              alt="Image of an Eisenhower Matrix"
+              layout="responsive"
+            />
+            <Link href="https://www.eisenhower.me/eisenhower-matrix/#:~:text=What%20is%20the%20Eisenhower%20Matrix,or%20not%20do%20at%20all.">
+              Learn More Here
+            </Link>
```
Comment (maxitect) [inline src/app/moods/info/page.tsx:101]: Possible to stylise this to look more like a link? Right now it looks like plain text and it is not obvious to me that it is a link. [will not stop merge if refused]

File: src/app/moods/info/page.tsx [hunk 2]
```diff
@@ -1,3 +1,164 @@
...
+              alt="Image of an Eisenhower Matrix"
+              layout="responsive"
+            />
+            <div className="flex flex-col gap-3">
+              <h2 className="text-xl font-semibold">The Eisenhower Matrix</h2>
+              <p>
+                The Eisenhower Matrix helps you prioritize tasks based on
+                urgency and importance. It divides tasks into four quadrants:
+              </p>
+            </div>
```
Comment (maxitect) [inline src/app/moods/info/page.tsx:72]:
```suggestion
            <div className="flex flex-col gap-3">
              <h2 className="text-xl font-semibold">The Eisenhower Matrix</h2>
              <p>
                The Eisenhower Matrix helps you prioritize tasks based on
                urgency and importance. It divides tasks into four quadrants:
              </p>
            </div>
            <Image
              src="/images/priorityCubeImage.png"
              width={600}
              height={600}
              alt="Image of an Eisenhower Matrix"
              layout="responsive"
            />
```
To be more consistent with with mood info page

File: src/app/moods/info/page.tsx [hunk 3]
```diff
@@ -1,3 +1,164 @@
...
+      <Header title="Learn More" isInfoPage={true} />
+      <Toggle
+        isToggled={isPriority}
+        setIsToggled={setIsPriority}
+        toggledOff={"Mood"}
+        toggledOn={"Priority"}
+        showLabels={true}
+      />
+      <div className="w-10/12 m-auto flex flex-col gap-10 mt-8">
+        {!isPriority ? (
```
Comment (maxitect) [inline src/app/moods/info/page.tsx:57]:
```suggestion
        {isPriority ? (
```
Currently the info pages are flipped

File: src/app/moods/loading.tsx
Comment (maxitect) [inline src/app/moods/loading.tsx:1]: I don't think this is working?

**Review by maxitect — CHANGES_REQUESTED** [review]
This is great, good work, love the re-use of the toggle too!

Couple of changes, main one being the information and toggle are currently flipped. Not sure the loading.tsx page is currently working as intended but we can look at this in another PR as it is not really the scope of this one.

Really like the refactoring of Header and the back button, this could also be a solution for the add tools page @JasonWarrenUK

File: src/app/moods/info/page.tsx (hunk 1 above)
Comment (jackcasstlesjones) [reply src/app/moods/info/page.tsx:101]: done

File: src/app/moods/info/page.tsx (hunk 2 above)
Comment (jackcasstlesjones) [reply src/app/moods/info/page.tsx:72]: It is currently in line with the figma design, I propose we leave it that way for now

File: src/app/moods/loading.tsx
Comment (jackcasstlesjones) [reply src/app/moods/loading.tsx:1]: No but we should leave it there for now

https://nextjs.org/learn/dashboard-app/streaming#streaming-a-whole-page-with-loadingtsx

File: src/app/moods/info/page.tsx (hunk 1 above)
Comment (maxitect) [reply src/app/moods/info/page.tsx:101]: Is it? xD

**Review by maxitect — DISMISSED** [review]

**Review by maxitect — APPROVED** [review]

**Review by JasonWarrenUK — APPROVED** [review]

**Review by maxitect — DISMISSED** [review]

**Review by maxitect — APPROVED** [review]

**Pull request merged**